layouts/_default/landing.html

```diff
@@ -0,0 +1,192 @@
+{{ define "body-class" }}
+  text-[16px] bg-beige font-serif leading-normal text-s-9
+{{ end }}
+{{ define "html-class" -}}
+  scroll-smooth
+{{- end }}
+{{ define "static-head" }}
+  {{ partialCached "header/static-tw.html" . }}
+{{ end }}
+
+{{ define "header" }}
+  <header data-ga-category="header">
+    {{ block "nav" . }}
+      {{ partialCached "tw/nav.html" . }}
+    {{ end }}
+    {{ block "banner" . }}
+      {{ if .Param "banner-active" }}
+        {{ partialCached "tw/banner.html" . }}
+      {{ end }}
+    {{ end }}
+    {{ block "topper" . }}
+      {{ if .Param "topper-active" }}
+        {{ partialCached "tw/promo-topper.html" . }}
+      {{ end }}
+    {{ end }}
+    {{ block "promo" . }}
+      {{ partialCached "tw/promo-header.html" . }}
+    {{ end }}
+  </header>
+{{ end }}
+
+{{ define "main" }}
+  <article class="bg-white px-4 pt-12 pb-2 md:pb-6 overflow-visible">
+    <header>
+      <div class="mx-auto w-full max-w-screen-md">
+        <h1
+          class="font-sans text-2xl sm:text-4xl md:text-5xl font-black leading-none tracking-wide"
```

Comment:
If you run `yarn format`, Prettier should arrange your Tailwind classes in some predefined order with the media query variants last.